doc/compiler/aot/SymbolValidationManager.md

```diff
@@ -119,7 +119,7 @@ validated; fail otherwise
 Class Chains are a core component of AOT validation; they ensure that
 the shape of a class is the same as that during the compile run. For
 technical details about Class Chains, see
-[AOT Class Chains](https://github.com/eclipse-openj9/openj9/blob/master/doc/compiler/aot/AOTClassChains.md).
+[AOT Class Chains](https://github.com/eclipse-openj9/openj9/blob/master/doc/compiler/aot/ClassChains.md).
```

Comment:
```suggestion
[AOT Class Chains](ClassChains.md).
```
Since you're fixing up the link, might as well make it a relative path so that it's consistent with other links in the docs, and so that it's more robust.